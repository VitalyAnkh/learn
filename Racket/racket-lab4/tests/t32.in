(intersect '(2 7 1 8) '(3 1 4))
(intersect '(to be) '(or not to be))
(intersect '(left right) '(right wrong))
(intersect '(2 7 1 8) (intersect '(2 8 1) (intersect '(3 1 4) '(1 5 9 2 6))))
