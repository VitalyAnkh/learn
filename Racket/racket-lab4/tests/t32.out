'(1)
'(to be)
'(right)
'(1)
